### ISSUE TITLE
Improve article 1310.5778

### ISSUE THREAD

**Issue by dginev** [dginev/ar5iv#62]
**Exact location of issue**

[Figure 7 of 1310.5778](https://ar5iv.org/html/1310.5778#S5.F7)

**Problem details**

There is an artifact subfigure inserted in the HTML markup. The first subfigure cell contains a single paragraph, with a single math element,  which holds a table with a single row, with two `mtd` cells, each of which containing a single mi with no content (or invisible content).

This leads to an empty subfigure panel, and shifts to the right the two "real" subfigure cells, leading to an off-center figure.

**Expected behavior**

All of this artifact markup can be pruned away in this context, or can be represented better if there was a good reason to generate _some_ markup, prescribed in the original TeX source. Figure 7 should look centered and the subfigures should take the appropriate amount of horizontal space.

**Comment by dginev** [comment]
The source here is something to admire:

```tex
\begin{figure}
$\begin{array}{cc}
\subfloat[]{\includegraphics[width=.45\columnwidth]{SingleBond-gkT0p3D.eps} \label{fig:Own_Dil} } &
\subfloat[]{\includegraphics[width=.45\columnwidth]{SingleBond-gkT0p3S.eps} \label{fig:Own_Sat} }
\end{array}$
\caption{...}
\label{fig:Own_SatAndDil}
\end{figure}
```

I suspect we would ideally want to unwind the `{array}` into subfigure XML markup...

**Comment by dginev** [comment]
Resolved after the latest update with latexml v0.8.8
